echo-pipelinetriggers/src/main/java/com/netflix/spinnaker/echo/pipelinetriggers/PipelineCache.java

```diff
@@ -170,7 +168,15 @@ void pollPipelineConfigs() {
         }
       })
       .filter(m -> !m.isEmpty())
-      .map(m -> objectMapper.convertValue(m, Pipeline.class))
+      .map(m -> {
+        try {
+          return objectMapper.convertValue(m, Pipeline.class);
+        } catch (Exception e) {
```

Comment:
Since there is another try/catch in this block, does it make sense to break this out into a helper `planV2Template` or something similar and wrap all the processing in a try/catch there for readability?

Reply:
Extracting to two functions for readability. I still want to keep the functions separate with their own try/catch, though, because I like that the function names explicitly state what they're going to do and handle errors in the way they want.

Reply:
:+1: